pkg/pipeline/conntrack/hash.go

```diff
@@ -97,11 +82,17 @@ func computeHashFields(flowLog config.GenericMap, fieldNames []string, hasher ha
 		}
 		bytes, err := toBytes(f)
 		if err != nil {
-			return nil, err
+			return 0, err
 		}
 		hasher.Write(bytes)
 	}
-	return hasher.Sum([]byte{}), nil
+	return hasher.Sum64(), nil
+}
+
+func uint64ToBytes(data uint64) []byte {
+	b := make([]byte, 8)
+	binary.BigEndian.PutUint64(b, data)
```

Comment:
I guess the encoding order (little vs big endian) does not matter here, right?

Reply:
right.